candecode_node:
  ros__parameters:
    decode_choices: false
    warn_if_unknown: false
    dbc_file: /Users/tfoldi/Developer/mobility/canedge-influxdb-writer/dbc_files/Model3CAN.dbc
    navsat_mapping:
      frame_id: "base_link"
      canbus_message_id: 79
      field_mapping:
        latitude: GPSLatitude04F
        longitude: GPSLongitude04F
        altitude: ""
        accuracy: GPSAccuracy04F
        status: ""
    twist_mapping:
      frame_id: "base_link"
      canbus_message_id: 599
      uom: "kph"
      field_mapping:
        linear:
          x: DI_vehicleSpeed
